documentation/sphinx/source/command-line-interface.rst

```diff
@@ -64,7 +64,7 @@ The ``commit`` command commits the current transaction. Any sets or clears execu
 configure
 ---------
 
-The ``configure`` command changes the database configuration. Its syntax is ``configure [new|tss] [single|double|triple|three_data_hall|three_datacenter] [ssd|memory] [grv_proxies=<N>] [commit_proxies=<N>] [resolvers=<N>] [logs=<N>] [count=<TSS_COUNT>] [perpetual_storage_wiggle=<WIGGLE_SPEED>] [perpetual_storage_wiggle_locality=<<LOCALITY_KEY>:<LOCALITY_VALUE>|0>] [storage_migration_type={disabled|aggressive|gradual}] [tenant_mode={disabled|optional_experimental|required_experimental}]``.
+The ``configure`` command changes the database configuration. Its syntax is ``configure [new|tss] [single|double|triple|three_data_hall|three_datacenter] [ssd|memory] [grv_proxies=<N>] [commit_proxies=<N>] [resolvers=<N>] [logs=<N>] [count=<TSS_COUNT>] [perpetual_storage_wiggle=<WIGGLE_SPEED>] [perpetual_storage_wiggle_locality=<<LOCALITY_KEY>:<LOCALITY_VALUE>|0>] [storage_migration_type={disabled|aggressive|gradual}] [tenant_mode={disabled|optional_experimental|required_experimental}] [encryption_data_at_rest_mode={aes_256_ctr|disabled}]``.
```

Comment:
Nit -- I think `data_at_rest_encryption_mode` sounds more natural than `encryption_data_at_rest_mode`. Just a thought to consider.

Reply:
I like `encryption_at_rest` or `encryption_at_rest_mode` if we need to include the `at_rest` component. `encryption_mode` could work too if that's not too ambiguous.

Reply:
I thought about `encryption_mode` , but it seems bit ambiguous, as there is already backup encryption implemented and not defining the scope is bit mis-leasing. For instance: in future it is possible to support in-memory encryption for any particular usecase OR for authorization, there is already an encryption concept for sharing Private keys. 

I would like to keep a longer name for the sake for non-ambiguity if possible, however, if someone feels strong about it then may be `encryption_at_rest_mode` seems like a better compromise. 

Thanks